apierror.h

```diff
@@ -68,6 +70,6 @@
 /*! \brief Helper method to get a string representation of an API error code
  * @param[in] error The API error code
  * @returns A string representation of the error code */
-const char *janus_get_api_error(int error);
+shared const char *janus_get_api_error(int error);
```

Comment:
I've noticed you've added this `shared` pretty much everywhere. What is it for? Is it just an explicit way of saying a method is not `static`? Why is this needed on Windows? Is it safe to have for OSX and Linux? Just asking as I'm personally not familiar with it.